auth0/src/main/java/com/auth0/android/authentication/ParameterBuilder.java

```diff
@@ -63,11 +70,22 @@
     public static final String SCOPE_OPENID = "openid";
     public static final String SCOPE_OFFLINE_ACCESS = "openid offline_access";
 
+    /**
+     * @deprecated The {@code id_token} parameter is only used when making requests to the legacy
```

Comment:
I'd link to https://auth0.com/docs/api/authentication#database-ad-ldap-active- as you did in others.

Also, DEVICE constant is still here. Should go away with ID token, same reason

Reply:
Pushed a new commit to add the link and also deprecate the `DEVICE` constant